adobe/spectrum-web-components
# fix(link): support "secondary" variant

## Description
Add support for the `secondary` variant of an `sp-link` element.

Update the documentation of link for fuller coverage of the element's features.

## Motivation and context
Full support for the Spectrum specification of a Link.

## How has this been tested?
- VRT expansion 
   1. Go to https://link-secondary--spectrum-web-components.netlify.app/components/link
   2. See the Secondary and Secondary Quiet link options

## Types of changes

## Checklist

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
